packages/dts-generator/src/download-apijson.ts

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+import { getLogger } from "@ui5/logger";
+const log = getLogger("@ui5/dts-generator/download-apijson");
+import esMain from "es-main";
+import _ from "lodash";
```

Comment:
```
import { pick } from "lodash";
```

as long as you don't need `_` anywhere else

Reply:
not possible because lodash is a commonjs module, see below